apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx

```diff
@@ -126,6 +143,34 @@ const NftViewer = ({ route }: Props) => {
 
   const nftCapabilities = useMemo(() => getNftCapabilities(nft), [nft]);
 
+  const [floorPriceLoading, setFloorPriceLoading] = useState(false);
+  const [ticker, setTicker] = useState("");
+  const [floorPrice, setFloorPrice] = useState(null);
+
+  useEffect(() => {
+    let mounted = true
+    if(mounted) {
+      setFloorPriceLoading(true);
+      getFloorPrice(nft, currency)
+        .then(
+          result => {
+            if (result) {
+              setTicker(result.ticker);
+              setFloorPrice(result.value);
+            }
+          },
+          err => {
+            throw new Error(err);
+          },
+        )
+        .catch(err => console.log("error", err))
```

Comment:
this is recovering the error which is not something you want I think. we also don't want to just console.log errors but either use them to display to the UI or just "ignore it". http errors already get logged in the logs (via the logger).